examples/2d/parameters_ec_performance_test.toml

```diff
@@ -1,57 +0,0 @@
-####################################################################################################
-# Simulation
```

Comment:
Correct. We introduced it some time ago for reproducible performance measurement. Since it's basically the same as `parameters_euler_ec.toml` with a bigber final time, there is no need to keep it around. Additionally, it's never used in CI tests.

Reply:
To be a bit more correct: I've removed the file completely.